src/core_seaice/Registry.xml

```diff
@@ -3994,12 +4004,12 @@
 		<var name="uAirVelocityVertex"			type="real"		dimensions="nVertices Time"					name_in_code="uAirVelocityVertex"			units="m/s"		packages="pkgBergs"/>
 		<var name="vAirVelocityVertex"			type="real"		dimensions="nVertices Time"					name_in_code="vAirVelocityVertex"			units="m/s"		packages="pkgBergs"/>
 		<var name="airDensityVertex"			type="real"		dimensions="nVertices Time"					name_in_code="airDensityVertex"				units="kg/m^3"	packages="pkgBergs"/>
-		<var name="bergAreaVertex"	type="real"		dimensions="nBergCategories nVertices Time"	name_in_code="bergAreaVertex"		units="m^2"		packages="pkgBergs"/>
-		<var name="bergLengthVertex"	type="real"		dimensions="nBergCategories nVertices Time"	name_in_code="bergLengthVertex"		units="m"		packages="pkgBergs"/>
-		<var name="bergCoefForIceU"	type="real"		dimensions="nBergCategories nVertices Time"	name_in_code="bergCoefForIceU"		units="kg*m/s^2"		packages="pkgBergs"/>
-		<var name="bergCoefForIceV"	type="real"		dimensions="nBergCategories nVertices Time"	name_in_code="bergCoefForIceV"		units="kg*m/s^2"		packages="pkgBergs"/>
-		<var name="bergStressU"	type="real"		dimensions="nVertices Time"	name_in_code="bergStressU"		units="kg*m/s^2"		packages="pkgBergs"/>
-		<var name="bergStressV"	type="real"		dimensions="nVertices Time"	name_in_code="bergStressV"		units="kg*m/s^2"		packages="pkgBergs"/>
+		<var name="bergAreaVertex"				type="real"		dimensions="nBergCategories nVertices Time"	name_in_code="bergAreaVertex"				units="m^2"		packages="pkgBergs"/>
```

Comment:
Note this are only formatting changes so that they line up in the text editor I was using - I can revert these if desired.

Reply:
So, we have a new convention in the Registry that we are SLLLLLLOOOOOOOWWWWWWWWLLLLLLLYYYYYY changing to....